src/skmultiflow/trees/hoeffding_tree_regressor.py

```diff
@@ -197,12 +197,12 @@ def normalize_sample(self, X):
         """
         normalized_sample = []
         for i in range(len(X)):
-            sd_squared = (self.sum_of_attribute_squares[i] - self.sum_of_attribute_values[i] ** 2
-                          / self.samples_seen) / self.samples_seen
-            if self.samples_seen > 1 and sd_squared >= 0:
+            var = ((self.sum_of_attribute_squares[i] - self.sum_of_attribute_values[i] ** 2
```

Comment:
`self.sum_of_attribute_values[i] * self.sum_of_attribute_values[i]` is faster than `self.sum_of_attribute_values[i] ** 2`

Reply:
Did not know that, dully noted. Thanks for the info!